### ISSUE TITLE
Updating a component in webpack-dev-server causes Uncaught TypeError

### ISSUE THREAD

**Issue by muddafugga** [guiqui/react-timeline-gantt#25]
When using react-timeline-gantt in webpack-dev-server, a hot reload triggers the following during rendering:

> index.js:1 Uncaught TypeError: Cannot assign to read only property 'backgroundColor' of object '#<Object>'
    at e.value (index.js:1)
    at e.value (index.js:1)
    at e.value (index.js:1)
    at e.value (index.js:1)
    at e.load (index.js:1)
    at new t (index.js:1)
    at constructClassInstance (react-dom.development.js:11361)
    at updateClassComponent (react-dom.development.js:14687)
    at beginWork (react-dom.development.js:15644)
    at performUnitOfWork (react-dom.development.js:19312)

**Comment by oliska** [comment]
I experienced same problem... caused by config object

**Comment by adback03** [comment]
Yup, started using this package recently -- same error here. On any re-render the error gets thrown. In my case, I'm using react router. When I change routes and go back to the page with the timeline component, this error appears.

**Comment by SharePointPro** [comment]
I'm also getting this issue.

**Comment by jonnoevans** [comment]
Also getting this issue
